lib/Doctrine/Migrations/Version/Version.php

```diff
@@ -4,237 +4,26 @@
 
 namespace Doctrine\Migrations\Version;
 
-use DateTimeImmutable;
-use DateTimeZone;
-use Doctrine\DBAL\Connection;
-use Doctrine\DBAL\Types\Type;
-use Doctrine\Migrations\AbstractMigration;
-use Doctrine\Migrations\Configuration\Configuration;
-use Doctrine\Migrations\Exception\MigrationNotConvertibleToSql;
-use Doctrine\Migrations\MigratorConfiguration;
-use Doctrine\Migrations\OutputWriter;
-use Doctrine\Migrations\Tracking\TableDefinition;
-use function assert;
-use function count;
-use function date_default_timezone_get;
-use function in_array;
-use function str_replace;
-
-/**
- * The Version class represents a single migration version. It wraps around your migration class
- * that extends the AbstractMigration class.
- *
- * @internal
- */
-class Version
+final class Version
 {
-    /** @var Configuration */
-    private $configuration;
-
-    /** @var OutputWriter */
-    private $outputWriter;
-
     /** @var string */
     private $version;
 
-    /** @var AbstractMigration */
-    private $migration;
-
-    /** @var Connection */
-    private $connection;
-
-    /** @var string */
-    private $class;
-
-    /** @var int */
-    private $state = State::NONE;
-
-    /** @var ExecutorInterface */
-    private $versionExecutor;
-
-    public function __construct(
-        Configuration $configuration,
-        string $version,
-        string $class,
-        ExecutorInterface $versionExecutor
-    ) {
-        $this->configuration   = $configuration;
-        $this->outputWriter    = $configuration->getOutputWriter();
-        $this->class           = $class;
-        $this->connection      = $configuration->getConnection();
-        $this->migration       = new $class($this);
-        $this->version         = $version;
-        $this->versionExecutor = $versionExecutor;
-    }
-
-    public function __toString() : string
+    public function __construct(string $version)
     {
-        return $this->version;
+        $this->version = $version;
     }
 
-    public function getVersion() : string
+    public function __toString()
```

Comment:
Hard to catch, but this removes the return type from the method. Was this on purpose?

From: https://github.com/lcobucci/doctrine-migrations/pull/1/commits/b4e07acfce8b6380504e67517d367583e69d420d.

Reply:
Will fix. The reason for it was a complain by php, or the method signature... but must be a mistake, looking at https://3v4l.org/DHAEs should work!